### ISSUE TITLE
tooltip duration property

### ISSUE THREAD

**Issue by jchamet** [sir-dunxalot/ember-tooltips#10]
It should be possible to set tooltips to close automatically after a certain duration

**Comment by sir-dunxalot** [comment]
#9 adds the ability for a tooltip's visibility to be bound to a boolean property instead of an event. Whilst this would add a bit of logic to the component or controller the boolean property is bound to (e.g. after 3 seconds, set `showTheTooltip` to false) it probably makes more sense than a tooltip duration property, which would interfere with events.

What use cases are you thinking of? And would #9 solve them?

**Comment by jchamet** [comment]
I am applying tooltips to input fields. I think tooltips are great for better explaining an input fields but when the user is typing, it doesn't make sense to continue showing the tooltip for the entire duration of focus, especially for large entries.

#9 would let me setup a duration and destroy tooltips after the duration, but I think this would be better managed as a property of the component.

**Comment by sir-dunxalot** [comment]
Ok, that seems reasonable.

So here, you would expect to do something like the following?

```hbs
{{input
  value=whatever
  tooltipContent='Fill me in'
  tooltipDuration=3000
  tooltipEvent='focus'
}}
```

**Comment by jchamet** [comment]
Yes! That would be perfect.

**Comment by sir-dunxalot** [comment]
Ok. I'll wait until #9 is merged before adding this feature.

**Comment by davidgovea** [comment]
Been using this recently - helps with the hover-triggered tooltips that somehow get stuck open. No way to fully avoid that, but if they auto-close after 5sec, you prevent UI eyesores.

**Comment by sir-dunxalot** [comment]
Awesome. I'm gonna get on this tomorrow (Saturday). Sorry for the delay.

**Comment by jchamet** [comment]
^^ I noticed that issue as well. The pull request looks good, I'm really looking forward to the merge!

**Comment by sir-dunxalot** [comment]
This should be resolved by #11 and `v0.4.0`. let me know if it doesn't add what you want.

**Comment by sir-dunxalot** [comment]
This seems like it's now supported. If not let me know and I will reopen this issue.
